pkg/resourcegroup/runaway/manager.go

```diff
@@ -176,11 +178,21 @@ func (rm *Manager) RunawayRecordFlushLoop() {
 			flushRunawayRecords()
 			fired = true
 		case r := <-recordCh:
-			records = append(records, r)
+			key := recordKey{
+				ResourceGroupName: r.ResourceGroupName,
+				SQLDigest:         r.SQLDigest,
+				PlanDigest:        r.PlanDigest,
+				Match:             r.Match,
```

Comment:
Should `Repeats` initialized with `1` so it means how many queries has been caught during that time?

Reply:
Indeed, maybe 1 is more intuitive, I was wrong about 0 as the default value.